# Chinese (Traditional) translation for linuxmint
# Copyright (c) 2014 Rosetta Contributors and Canonical Ltd 2014
# This file is distributed under the same license as the linuxmint package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2014.
#
msgid ""
msgstr ""
"Project-Id-Version: linuxmint\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2023-11-27 14:33+0000\n"
"PO-Revision-Date: 2020-04-23 16:34+0000\n"
"Last-Translator: H.G. <Unknown>\n"
"Language-Team: Chinese (Traditional) <zh_TW@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2023-12-03 13:31+0000\n"
"X-Generator: Launchpad (build 886068b1e1d10de444fbd53b0c7d3f6d4fb3e658)\n"

#: lib/mintstick.py:312
msgid "The USB stick was formatted successfully."
msgstr "成功格式化 USB 隨身碟。"

#: lib/mintstick.py:315
#, python-format
msgid "An error occured while creating a partition on %s."
msgstr "在 %s 建立磁碟分割區時發生錯誤。"

#: lib/mintstick.py:317 lib/mintstick.py:427
msgid "Authentication Error."
msgstr "驗證錯誤。"

#: lib/mintstick.py:322 lib/mintstick.py:433
msgid "An error occurred."
msgstr "發生錯誤。"

#: lib/mintstick.py:420
msgid "The image was successfully written."
msgstr "成功寫入映像檔。"

#: lib/mintstick.py:423
msgid "Not enough space on the USB stick."
msgstr "USB 隨身碟空間不足。"

#: lib/mintstick.py:425
msgid "An error occured while copying the image."
msgstr "複製映像檔時發生錯誤。"

#: lib/verify.py:54
msgid "bytes"
msgstr ""

#: lib/verify.py:54
msgid "KB"
msgstr ""

#: lib/verify.py:54
msgid "MB"
msgstr ""

#: lib/verify.py:54
msgid "GB"
msgstr ""

#: lib/verify.py:54
msgid "TB"
msgstr ""

#: lib/verify.py:87
msgid "ISO images"
msgstr ""

#: lib/verify.py:91
msgid "Checksum files"
msgstr ""

#: lib/verify.py:95
msgid "GPG signatures"
msgstr ""

#: lib/verify.py:107 lib/verify.py:108 lib/verify.py:109
msgid "Calculating..."
msgstr ""

#: lib/verify.py:167
msgid "No volume ID found"
msgstr ""

#: lib/verify.py:208
msgid "The checksum is correct"
msgstr ""

#: lib/verify.py:209
msgid ""
"The checksum is correct but the authenticity of the sum was not verified."
msgstr ""

#: lib/verify.py:211
msgid "Checksum mismatch"
msgstr ""

#: lib/verify.py:212
msgid "Download the ISO image again. Its checksum does not match."
msgstr ""

#: lib/verify.py:223
msgid "The sums file could not be downloaded. Check the URL."
msgstr ""

#: lib/verify.py:232
msgid "The gpg file could not be downloaded. Check the URL."
msgstr ""

#: lib/verify.py:243
msgid "The sums file could not be checked."
msgstr ""

#: lib/verify.py:250
msgid "The gpg file could not be checked."
msgstr ""

#: lib/verify.py:260
msgid "Integrity check failed"
msgstr ""

#: lib/verify.py:269
msgid "The SHA256 sums file is not signed."
msgstr ""

#: lib/verify.py:273
#, python-format
msgid "Signed by: %s"
msgstr ""

#: lib/verify.py:287
msgid "Unknown signature"
msgstr ""

#: lib/verify.py:288
msgid "Key not found on keyserver."
msgstr ""

#: lib/verify.py:299 lib/verify.py:305
msgid "Everything looks good!"
msgstr ""

#: lib/verify.py:300
msgid "This is an official ISO image."
msgstr ""

#: lib/verify.py:306
msgid "This ISO image is verified by a trusted signature."
msgstr ""

#: lib/verify.py:309
msgid "If you trust the signature you can trust the ISO."
msgstr ""

#: lib/verify.py:310
msgid "Untrusted signature"
msgstr ""

#: lib/verify.py:311
msgid "This ISO image is verified by an untrusted signature."
msgstr ""

#: lib/verify.py:316
msgid "An error occurred"
msgstr ""

#: lib/verify.py:327
msgid "The SHA256 sum of the ISO image is incorrect."
msgstr ""

#: lib/verify.py:328
msgid "The SHA256 sums file does not contain sums for this ISO image."
msgstr ""

#: generate_additional_files.py:22 generate_additional_files.py:34
#: share/mintstick/mintstick.ui.h:38
msgid "USB Image Writer"
msgstr "USB 映像寫入工具"

#: generate_additional_files.py:22 generate_additional_files.py:34
#: generate_additional_files.py:67 share/mintstick/mintstick.ui.h:39
msgid "Make a bootable USB stick"
msgstr "製作可開機 USB 隨身碟"

#: generate_additional_files.py:46 generate_additional_files.py:58
#: share/mintstick/mintstick.ui.h:27
msgid "USB Stick Formatter"
msgstr "USB 隨身碟格式化程式"

#: generate_additional_files.py:46 generate_additional_files.py:58
#: generate_additional_files.py:77 share/mintstick/mintstick.ui.h:28
msgid "Format a USB stick"
msgstr "格式化 USB 隨身碟"

#: generate_additional_files.py:67
msgid "Make bootable USB stick"
msgstr "製作可開機 USB 隨身碟"

#: generate_additional_files.py:77 share/mintstick/mintstick.ui.h:26
msgid "Format"
msgstr "格式化"

#: generate_additional_files.py:86 share/mintstick/mintstick.ui.h:32
#: share/mintstick/verify.ui.h:7
msgid "Verify"
msgstr ""

#: generate_additional_files.py:86 share/mintstick/mintstick.ui.h:33
msgid "Verify the authenticity and integrity of the image"
msgstr ""

#: generate_additional_files.py:115 generate_additional_files.py:144
msgid ""
"This will destroy all data on the USB stick, are you sure you want to "
"proceed?"
msgstr "這會銷毀 USB 隨身碟裡的所有資料，您是否確定要繼續?"

#: share/mintstick/mintstick.ui.h:1
msgid "USB stick:"
msgstr "USB 隨身碟:"

#: share/mintstick/mintstick.ui.h:2
msgid "Filesystem:"
msgstr ""

#: share/mintstick/mintstick.ui.h:3
msgid "Select a USB stick"
msgstr "選擇 USB 隨身碟"

#: share/mintstick/mintstick.ui.h:4
msgid ""
"FAT32 \n"
"  + Compatible everywhere.\n"
"  -  Cannot handle files larger than 4GB.\n"
"\n"
"exFAT\n"
"  + Compatible almost everywhere.\n"
"  + Can handle files larger than 4GB.\n"
"  -  Not as compatible as FAT32.\n"
"\n"
"NTFS \n"
"  + Compatible with Windows.\n"
"  -  Not compatible with Mac and most hardware devices.\n"
"  -  Occasional compatibility problems with Linux (NTFS is proprietary and "
"reverse engineered).\n"
"\n"
"EXT4 \n"
"  + Modern, stable, fast, journalized.\n"
"  + Supports Linux file permissions.\n"
"  -  Not compatible with Windows, Mac and most hardware devices.\n"
msgstr ""
"FAT32 \n"
"  + 相容性最佳。\n"
"  -  無法處理大於 4GB 的檔案。\n"
"\n"
"exFAT\n"
"  + 幾乎每個平臺都可以相容。\n"
"  + 無法處理大於 4GB 的檔案。\n"
"  -  相容性比 FAT32 稍差。\n"
"\n"
"NTFS \n"
"  + 相容於 Windows。\n"
"  -  不相容 Mac 與大多數硬體裝置。\n"
"  -  在 Linux 下偶有相容性問題 (因 NTFS 是專有格式，遂以逆向工程完成)。\n"
"\n"
"EXT4 \n"
"  + 現代化、穩定、快速、具日誌功能。\n"
"  + 支援 Linux 檔案權限。\n"
"  -  不相容於 Windows、Mac 與大多數硬體裝置。\n"

#: share/mintstick/mintstick.ui.h:23
msgid "Volume label:"
msgstr "儲存區標籤:"

#: share/mintstick/mintstick.ui.h:24
msgid "Choose a name for your USB Stick"
msgstr "命名您的 USB 隨身碟"

#: share/mintstick/mintstick.ui.h:25
msgid "USB Stick"
msgstr "USB 隨身碟"

#: share/mintstick/mintstick.ui.h:29
msgid "ISO image:"
msgstr ""

#: share/mintstick/mintstick.ui.h:30
msgid "Select an image"
msgstr "選擇映像檔"

#: share/mintstick/mintstick.ui.h:31
msgid "Select Image"
msgstr "選擇映像檔"

#: share/mintstick/mintstick.ui.h:34
msgid "Write"
msgstr "寫入"

#: share/mintstick/mintstick.ui.h:35
msgid "This ISO looks like a Windows image."
msgstr ""

#: share/mintstick/mintstick.ui.h:36
msgid "Windows images require special processing."
msgstr ""

#: share/mintstick/mintstick.ui.h:37
msgid "More information"
msgstr ""

#: share/mintstick/verify.ui.h:1
msgid "ISO:"
msgstr ""

#: share/mintstick/verify.ui.h:2
msgid "Volume:"
msgstr ""

#: share/mintstick/verify.ui.h:3
msgid "SHA256sum:"
msgstr ""

#: share/mintstick/verify.ui.h:4
msgid "Size:"
msgstr ""

#: share/mintstick/verify.ui.h:5
msgid "SHA256 sums file:"
msgstr ""

#: share/mintstick/verify.ui.h:6
msgid "GPG signed file:"
msgstr ""

#: share/mintstick/verify.ui.h:8
msgid "URLs"
msgstr ""

#: share/mintstick/verify.ui.h:9
msgid "SHA256 sums file"
msgstr ""

#: share/mintstick/verify.ui.h:10
msgid "GPG signed file"
msgstr ""

#: share/mintstick/verify.ui.h:11
msgid "Local files"
msgstr ""

#: share/mintstick/verify.ui.h:12
msgid "SHA256 sum"
msgstr ""

#: share/mintstick/verify.ui.h:13
msgid "Check"
msgstr ""

#: share/mintstick/verify.ui.h:14
msgid "Checksum"
msgstr ""

#: share/mintstick/verify.ui.h:15
msgid "Go back"
msgstr ""

#: share/mintstick/verify.ui.h:16
msgid "ISO Verification"
msgstr ""
